staff/edit-courses/delete-course.php

```diff
@@ -0,0 +1,17 @@
+<?php
+	require_once "../../common.php";
+    require_staff();
+
+    if (array_key_exists('course_code', $_GET)) {
+        $course_code = $_GET["course_code"]; // TODO[reece]: Allow courses to share a course_code
```

Comment:
Actually I don't think this will be an issue (thankfully). For example, I found out the new data structures course will be EECS 330, not EECS 360.